src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -102,6 +102,10 @@ public async Task ExecuteAsync(string stackName, bool saveCdkProject)
 
             var deployedApplication = deployedApplications.FirstOrDefault(x => string.Equals(x.Name, cloudApplicationName));
 
+            // Throw an error if we are NOT performing a redeployment and the DisableInteractive flag is set to true.
+            if (_toolInteractiveService.DisableInteractive && deployedApplication == null)
+                throw new InvalidCliArgumentException("The \"--disable-interactive\" CLI argument can only be used during a redeployment. Please remove this argument and try again.");

```

Comment:
We don't need this. We shouldn't blow up if the user is doing a deployment.

Reply:
As per my discussion with @normj, this flag should only be used if the user is trying to perform a redeployment. If a user is performing a new deployment with this flag, then an error must be thrown

Reply:
When this work is merged with Phil work to pass in a user settings file then the flag can be used for initial deployment as well as redeployments.

Reply:
Removed this line so that a user can perform an initial deployment even when this flag is set.